src/main/java/com/thoughtworks/gauge/FileHelper.java

```diff
@@ -82,4 +83,14 @@ public static File getDefaultImplFileName(String suffix, int count) {
     private static String getDefaultStepImplDir() {
         return getAbsolutePath(DEFAULT_SRC_DIR).toString();
     }
+    private static List<String> getDefaultStepImplDirs() {
+        List<String> dirs = new ArrayList<>();
+        for (String dir : DEFAULT_SRC_DIRS.split(",")) {
```

Comment:
```suggestion
        for (String dir : DEFAULT_SRC_DIRS) {
```

After declaring DEFAULT_SRC_DIRS as an array